feat(core/pane-layout): provide close/open event and default slot (#1563)
siemens/ix: .changeset/seven-coins-fly.md

---

> '@siemens/ix-angular': patch
> '@siemens/ix': patch
> '@siemens/ix-vue': patch

Reviewer: ```suggestion
'@siemens/ix-angular': minor
'@siemens/ix': minor
'@siemens/ix-vue': minor
```

---

> feat(core/pane-layout): provide close/open event and default slot #1563

Reviewer 2: Adapt changeset to new format

> Update expandedChange event to trigger only on user interactions and refine default slot behavior in ix-pane-layout.

Reviewer: ```suggestion
Update expandedChange event to trigger only on user interactions and add unnamed default slot for ix-pane-layout content.
```
